docs/content/about-iota/iota-architecture/consensus.mdx

```diff
@@ -5,15 +5,40 @@ description: Overview of the IOTA consensus.
 tags: [core-protocol]
 ---
 
-The basic purpose of consensus in blockchains is to agree on a consistent order and ensure the availability of transactions.
+The primary goal of consensus in distributed systems is to establish a consistent order of transactions and ensure their availability. In IOTA, consensus is achieved through a combination of Delegated Proof-of-Stake (dPoS) and the Mysticeti Byzantine Fault Tolerant (BFT) consensus protocol.
 
-On IOTA, consensus has a simple API: validators submit different user transactions to consensus concurrently, and the consensus outputs a consistent stream of transactions across all well-behaving validators.
+IOTA's consensus mechanism ensures that:
 
-Byzantine-fault tolerant (BFT) consensus protocols are a rich area of research. The next-generation consensus engine in IOTA is based on the **Mysticeti** protocol.
-The protocol optimizes for both low latency and high throughput because:
-- It allows multiple validators to propose blocks in parallel, utilizing the full bandwidth of the network and providing censorship resistance. These are features of the DAG-based consensus protocols.
-- It takes only three rounds of messages to commit blocks from the DAGs, same as pBFT and matches the theoretical minimum.
-- The commit rule allows voting and certifying leaders on blocks in parallel, further reducing the median and tail latencies.
-- The commit rule also tolerates unavailable leaders without significantly increasing the commit latencies.
+- Users submit transactions to validators.
+- Validators execute the Mysticeti protocol to reach agreement on the order and finalization of transactions.
+- The result is a consistently ordered ledger that is agreed upon by an honest majority of validators.
```

Comment:
Weak arguments for this list. The first two items describe how consensus works, the last item is the actual protocol feature/guarantee. Maybe we should shift focus to features such as: lower latency, higher throughput, faster finality, more energy-efficient (compared to PoW), etc. Or, if we want to talk about protocol security guarantees, then we should be more formal: security, scalability, decentralization and how and why it resists known attacks/issues. Otherwise, this list is non-informative and confusing.